addons/website/static/src/js/website.seo.js

```diff
@@ -64,29 +66,38 @@
         refresh: function () {
             var self = this;
             self.$el.append("Loading...");
-            function addSuggestions (list) {
-                self.$el.empty();
-                // TODO Improve algorithm + Ajust based on custom user keywords
-                var regex = new RegExp(self.root, "gi");
-                var cleanList = _.map(list, function (word) {
-                    return word.replace(regex, "").trim();
-                });
-                // TODO Order properly ?
-                _.each(_.uniq(cleanList), function (keyword) {
-                    if (keyword) {
-                        var suggestion = new website.seo.Suggestion(self, {
-                            root: self.root,
-                            keyword: keyword,
-                            page: self.htmlPage,
-                        });
-                        suggestion.on('selected', self, function (word) {
-                            self.trigger('selected', word);
-                        });
-                        suggestion.appendTo(self.$el);
-                    }
-                });
-            }
-            $.getJSON("/website/seo_suggest/" + encodeURIComponent(this.root + " "), addSuggestions);
+            var language = self.language || website.get_context().lang.toLowerCase();
+            openerp.jsonRpc('/website/seo_suggest', 'call', {
+                'keywords': self.root,
+                'lang': language,
+            }).then(function(list){
+                self.addSuggestions(list);
+            });
+        },
+        addSuggestions: function(list) {
+            list  = JSON.parse(list);
+            var self = this;
+            self.$el.empty();
+            // TODO Improve algorithm + Ajust based on custom user keywords
+            var regex = new RegExp(self.root, "gi");
+            var cleanList = _.map(list, function (word) {
+                return word.replace(regex, "").trim();
+            });
+            // TODO Order properly ?
+            _.each(_.uniq(cleanList), function (keyword) {
+                if (keyword) {
+                    var suggestion = new website.seo.Suggestion(self, {
```

Comment:
keyword list contain this root keyword, and replace a root keyword to null string so need to check condition for keyword.